README.md

```diff
@@ -1,4 +1,5 @@
 # Happy Thoughts
+(OBS! This project is under construction!)
```

Comment:
First of all, Maria, I know you've heard this before from me but I just want to say again that you are EXTREMELY BRAVE and it's so amazing how you are so hardy and don't give up. I think, even though this is under construction, this project is showing your great quality as a developer.